spec/backupsss/tar_spec.rb

```diff
@@ -63,6 +63,22 @@
     end
   end
 
+  describe '#tar_command' do
+    context 'compress_archive true' do
+      subject { Backupsss::Tar.new('some/src', 'some/dest.tar').tar_command }
+
+      it { is_expected.to eq('tar -zcvf') }
+    end
+
+    context 'compress_archive false' do
+      subject do
```

Comment:
The first one, on line 68 is so much more pretty. Could you make this one look like that one. Use "{...}" here rather than "do...end"